src/services/app/client.ts

```diff
@@ -183,8 +184,8 @@ export class Client {
       const json = await http({
         context: { ctx },
         method: "POST",
-        url: `https://${subdomain}.${config.domains.app}${this.endpoint}`,
-        headers: { cookie, "x-gadget-environment": ctx.env.name },
+        url: `https://${subdomain}.${config.domains.app}/edit/api/graphql`,
```

Comment:
```suggestion
        url: `https://${subdomain}.${config.domains.app}${this.endpoint}`,
```

The `Client` has to use `this.endpoint` because it's used for both the [`Edit`](https://github.com/gadget-inc/ggt/blob/main/src/services/app/edit/edit.ts#L24) and [`Api`](https://github.com/gadget-inc/ggt/blob/main/src/services/app/api/api.ts#L24) clients.

If the `Api` client isn't supposed to use auth tokens, I'd suggest moving this logic into the `Edit` client.